File: components/sound/src/test/java/org/storm/sound/types/MediaSoundTest.java
package org.storm.sound.types;

import javafx.application.Application;
import javafx.stage.Stage;
import org.storm.sound.Sound;

public class MediaSoundTest extends Application {

  private static final String BGM_FILE = "src/test/resources/bgm.mp3";

  private Sound bgm;

  @Override
  public void start(Stage stage) {
    bgm = new MediaSound(BGM_FILE, false);
    bgm.adjustVolume(0.1);
    bgm.play();
  }

  public static void main(String[] args) {
    launch(args);
  }
}

File: components/sound/src/test/kotlin/org/storm/sound/types/MediaSoundTest.kt
package org.storm.sound.types

import javafx.application.Application
import javafx.stage.Stage
import org.storm.sound.Sound

class MediaSoundTest : Application() {

    companion object {
        private const val BGM_FILE = "bgm.mp3"
    }

    private lateinit var bgm: Sound

    override fun start(stage: Stage) {
        bgm = MediaSound(BGM_FILE, resource = true)
        bgm.adjustVolume(0.1)
        bgm.play()
    }

}
